setup.py

```diff
@@ -31,7 +31,7 @@
     entry_points={
         'console_scripts': ['gtfs-rt-validator-api=gtfs_rt_validator_api:main'],
     },
-    install_requires=["argh", "gcsfs"],
+    install_requires=["argh", "gcsfs", "pyarrow"],
```

Comment:
What is `pyarrow` for? I don't see it imported anywhere.

Reply:
it's used in the parquet loading, but imported internally by pandas 

https://github.com/cal-itp/gtfs-rt-validator-api/pull/4/files/3545dc543206595a23f7be9bb1db509f2709ab84#diff-b1389d2f9a2e1ac4ce431fa0f5d7709d688c11f306fa15cc88bf9a790c7cf1b3R158